templates/ords-remix-jwt-sample/ords/migrateScripts/protectEndpoints.js

```diff
@@ -50,13 +50,6 @@ async function protectEndpoints(
     'concert_app.adminuser.v1',
   )}
 
-  ${definePrivilege(
-    'concert_app_euser',
-    'Non authenticated end user privilege',
-    'Provides limited access to the concert app endpoints',
-    'concert_app.euser.v1',
-  )}
-
```

Comment:
Since we are dropping this privilege could you please:

1. Update the [ORDS Concert App - Securing Services](https://github.com/oracle/create-database-app/blob/main/templates/ords-remix-jwt-sample/ords/security/README.MD) documentation, this includes: 

- [Protecting our API Endpoints](https://github.com/oracle/create-database-app/blob/main/templates/ords-remix-jwt-sample/ords/security/README.MD#protecting-our-api-endpoints) Explain that the `concert_app.euser.v1` module is not protected. 
-  [Third Step: Grant permissions to the recently created API.](https://github.com/oracle/create-database-app/blob/main/templates/ords-remix-jwt-sample/ords/security/README.MD#third-step-grant-permissions-to-the-recently-created-api) Remove the `concert_app_euser` privilege from the table.

2. In the [ORDS Concert App - Defining and Creating the RESTful services](https://github.com/oracle/create-database-app/blob/main/templates/ords-remix-jwt-sample/ords/modules/README.md#ords-concert-app---defining-and-creating-the-restful-services) add a section to explain that the `concert_app.euser.v1` module is public.

3. In the [auth0Strategy function](https://github.com/oracle/create-database-app/blob/597e88ed6454f6450b7a231ce490b22af36140cb/templates/ords-remix-jwt-sample/app/utils/auth.server.ts#L41), remove the `concert_app_euser` scope. 
4. In the concert_app.euser.v1.sql module export, https://github.com/oracle/create-database-app/blob/597e88ed6454f6450b7a231ce490b22af36140cb/templates/ords-remix-jwt-sample/Database/Modules/concert_app.euser.v1.sql#L406 remove the `concert_app_euser` privilege (please export the new module definition and use it to update this document). 
5. In the concert_sample_app.sql export https://github.com/oracle/create-database-app/blob/597e88ed6454f6450b7a231ce490b22af36140cb/templates/ords-remix-jwt-sample/Database/Modules/CONCERT_SAMPLE_APP.sql#L1815, remove the `concert_app_euser` privilege (please export the definition and use it to update this document). 
6. In the openAPI catalog of the euser module (https://github.com/oracle/create-database-app/blob/main/templates/ords-remix-jwt-sample/ords/openAPI%20Catalog/concert_app.euser.v1.json), update the openAPI catalog with the new definition of the `euser` module.

Reply:
Changes requested in this thread will be handled in a separated PR.